.github/workflows/go-build.yml

```
@@ -420,51 +433,142 @@ jobs:
+      - name: Create output directory
+        run: mkdir -p build/out
+
+      - name: Write App Store Connect API key
+        env:
+          APP_STORE_CONNECT_KEY: ${{ secrets.APP_STORE_CONNECT_KEY }}
+        run: echo "$APP_STORE_CONNECT_KEY" > app_store_connect_key.json
 
-          # Replace version in Info.plist file
-          sed -i '' "s/<string>1.0.0<\/string>/<string>${VERSION}<\/string>/" "${APP}/Info.plist"
+      - name: Set up AMD64 environment
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:11:28: Double quote to prevent globbing and word splitting

(shellcheck)